storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/SampleApp.java

```diff
@@ -31,8 +25,28 @@ public static SampleApp get(@NonNull Context context) {
         return (SampleApp) context.getApplicationContext();
     }
 
+    // When another process of the app created (for example, for ContentProvider), onCreate() method of application object won't be called
+    // so we can't be sure, that it will be initialized
```

Comment:
Yep, I catched null pointer here in Sample App's ContentProvider

вс, 12 апр. 2015, 0:11, Dmitrii Nikitin <notifications@github.com>:

> In
> storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/SampleApp.java
> <https://github.com/pushtorefresh/storio/pull/275#discussion_r28198439>:
>
> > @@ -31,8 +25,28 @@ public static SampleApp get(@NonNull Context context) {
> >          return (SampleApp) context.getApplicationContext();
> >      }
> >
> > +    // When another process of the app created (for example, for ContentProvider), onCreate() method of application object won't be called
> > +    // so we can't be sure, that it will be initialized
>
> wow
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pushtorefresh/storio/pull/275/files#r28198439>.
>